config.ru

```diff
@@ -0,0 +1,7 @@
+require "bundler/setup"
+
+require_relative "./store"
+
+use Rack::MethodOverride
```

Comment:
no, just no reason to reinvent sth you have for free from framework (but those sinatra command do extactly the same)